### PR TITLE
Add class ball modularized and its unit test.

### PR DESCRIPTION
**Pull request by edwinPato**
We modularized class ball with baby steps and implemented its unit test.

### CODE REVIEW

**Review by Milena-Rodriguez-Montecinos — APPROVED** [review]

**Review by jchoque87 — APPROVED** [review]

**Comment by J0c4** [comment]
Nice PR!

**Review by serqiiiio — APPROVED** [review]

**Review by virichino — APPROVED** [review]

File: v2/test/ball.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Ball } from '../ball.js'
+
+test('Unit test with mocks', () => {
```
Comment (J0c4) [inline v2/test/ball.test.js]: Maybe this test description could be refined, it looks like it's not just "unit test with mocks", but verifies something else with the status of ball object

Comment (edwinPato) [reply v2/test/ball.test.js]: Ok! I am going to refine the test description. The function return a value set in the constructor of class ball.

Comment (J0c4) [reply v2/test/ball.test.js]: Thanks

**Pull request merged**